### PR TITLE
Format repository: Alex

### LINKED ISSUE #108: Organise repo into discrete folders
As a developer, I want to organize the project repository into discrete, well-structured folders, so that the codebase is easy to navigate, maintain, and scale, ensuring that future contributors can quickly locate and work on specific parts of the project without confusion.

Create the following folders:
- Astronomy API pipeline
- Aurorawatch API pipeline
- Quadhoral OpenMeteo pipeline
- Weekly OpenMeteo Pipeline
- SMS checker

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #108 #109 #110 #111 #112 #113 

What I have done:
- Created folders for each element of the architecture diagram
- Added a readme with a brief overview to:
1. Astronomy pipeline
2. Aurorawatch pipeline
3. Quadhoral openmeteo pipeline
4. Weekly openmeteo pipeline
5. SMS-Checker

### CODE REVIEW

File: astronomy-pipeline/README.md
Comment (CalebMathers) [inline astronomy-pipeline/README.md:1]: Lovely readme, keeps in format with the other readme files created earlier

File: aurorawatch-pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Aurorawatch Pipeline
+This folder contains the code for the Aurorawatch data pipeline. This includes `extract.py`, `transform.py`, `load.py` and `pipeline.py`. This pipeline uses the `requests` library to send HTML GET requests to the [Aurorawatch UK API](https://aurorawatch.lancs.ac.uk/api-info/0.2/), the `xml` library to parse and transform the received data and finally the `psycopg2` library to load this data onto our database.
```
Comment (CalebMathers) [inline aurorawatch-pipeline/README.md:2]: Love the link to the API

File: weekly-openmeteo/README.md
Comment (CalebMathers) [inline weekly-openmeteo/README.md:1]: Great explanation in all these files of what is intended

**Review by CalebMathers — APPROVED** [review]

File: astronomy-pipeline/README.md
Comment (MurasatoNaoya) [reply astronomy-pipeline/README.md:1]: Nice format using markdown '#'s for clear segmentation. Sets a clear standard for all other future READMEs in the StarWatch project.

File: aurorawatch-pipeline/README.md (hunk 1 above)
Comment (MurasatoNaoya) [reply aurorawatch-pipeline/README.md:2]: Good direct link to the AuroraWatch API, so the person looking at this README can directly look at the initial format of aurora data for themselves, and better appreciate how we've used it for this project.

File: quadhoral-openmeteo/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Quadhoral OpenMeteo Pipeline
+This folder contains the code for the [OpenMeteo API](https://open-meteo.com/en/docs) data pipeline. This includes `extract.py`, `transform.py`, `load.py` and `pipeline.py`. This pipeline uses the `requests` library to send HTML GET requests to the `OpenMeteo API` and the `psycopg2` library to load this data onto our database. The data requested includes `cloud cover`, `visibility` and `chance of precipitation`, this pipeline is therefore designed to run every four hours to ensure accurate data.
```
Comment (MurasatoNaoya) [inline quadhoral-openmeteo/README.md:2]: Good link to the OpenMeteo API for the reasons mentioned above.

File: astronomy-pipeline/README.md
Comment (olliejth) [inline astronomy-pipeline/README.md:1]: Add emojis to readme titles?

File: aurorawatch-pipeline/README.md
Comment (olliejth) [inline aurorawatch-pipeline/README.md:1]: Good example of the level of descriptiveness for the reames generally.

File: sms-checker/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# SMS-Checker Pipeline
+This folder contains the code for the SMS-checker application. This application can be triggered to query the database for any subscribers that are due an alert for a celestial body or aurora, this is done using the `psycopg2` library. These subscriber details are then used to send emails and SMS messages with the appropriate alerts using `boto3`.
```
Comment (MurasatoNaoya) [inline sms-checker/README.md:2]: Good discussion around the later functionality around what groups of subscribers should be notified about visible celestial bodies and/or events are near them. We need to keep this presently in mind so we build our earlier functionality around this key feature.

**Review by MurasatoNaoya — APPROVED** [review]

**Review by olliejth — APPROVED** [review]
Generally robust formatting for READMEs, I will die on the hill of including emojis but I suppose that can wait.

File: aurorawatch-pipeline/README.md (hunk 1 above)
Comment (AlexOBarnes) [reply aurorawatch-pipeline/README.md:2]: I am profoundly indebted to you for the unparalleled graciousness of your most discerning observations, which I shall treasure with the utmost reverence and gratitude, as they are indeed a testament to your impeccable wisdom and refinement

**Pull request merged**